src/plexil-adapter/README.md

```diff
@@ -8,20 +8,3 @@ Contents
 This directory contains C++ classes and code implementing the interface between
 the PLEXIL plans (found in ../plans) and the external world (lander,
 environment), as well as the ROS autonomy node.
-
-
-ROS Action prototype
---------------------
-
-This directory also contains a dummy stand-in ROS action server for GuardedMove,
-as well as a standalone node for testing it.  (GuardedMoveServer,
-GuardedMoveClient).  Run the following sequence, in three separate shells.  Note
-that the latter two executables are found in the `devel/lib/ow_autonomy` directory
-of your OceanWATERS workspace.
-
-
-```bash
-  roscore
-  GuardedMoveServer
-  GuardedMoveClient
-  ```
```

Comment:
You removed the old interfaces but shouldn't you put notes on the new ROS action interface?

Reply:
I removed the description of the dummy action prototype, but don't see a need to say anything about the ROS implementation or lander interface in this file -- these are documented in the user guide.